src/Application/Cli/TravisAutoDeployCommand.php

```diff
@@ -93,23 +93,23 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if ($travisBranch !== 'master') {
             $output->writeln('<comment>[NOT DEPLOYED] Deploying Couscous only for master branch</comment>');
 
-            return 0;
+            return 1;
```

Comment:
This is a behavior change, not sure we want that. Why change that? (same for the lines below)

Reply:
Is it really a behavior change?
The return values have been added in the https://github.com/CouscousPHP/Couscous/pull/259 PR.
Also it means the command has failed (`EXIT_FAILURE`), isn't it better? (can be useful when chaining commands in Bash for instance).

Reply:
When I added them I used `0` as the output only contains comments and no errors (and because that's the default value set by Symfony).

As far as I understand how the CI is configured this exit code won't change the behaviour of the CI as the command is run on `after_success`

Reply:
IMO, the comments look like errors, and that's why I changed the return codes, but I could be wrong.